Sources/PostmarkClient/Provider.swift

```diff
@@ -0,0 +1,47 @@
+import Vapor
+import HTTP
+
+
+public final class Provider: Vapor.Provider {
```

Comment:
This class can now be removed wholesale. The `PostmarkClient` will need to implement the new required method `static func configure(_ config: Config) throws` which is where you bring in your auth token configuration.